src/main/java/dansplugins/factionsystem/commands/abs/TabCompleterBase.java

```diff
@@ -0,0 +1,588 @@
+package dansplugins.factionsystem.commands.abs;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import dansplugins.factionsystem.data.PersistentData;
+import dansplugins.factionsystem.objects.domain.Faction;
+import dansplugins.factionsystem.services.ConfigService;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.TabCompleter;
+import org.bukkit.entity.Player;
+
+import javax.annotation.Nullable;
+
+public class TabCompleterBase implements TabCompleter {
+
+	PersistentData persistentData;
```

Comment:
These fields can be private

Reply:
These fields have been designated private.